test/docker/_pinger

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
```

Comment:
these files do not have `.js` extensions to avoid being grepped by lab in normal `npm test`